e2e_test/batch/basic/func.slt.part

```diff
@@ -504,10 +528,10 @@ select regexp_replace('abc123', 'abc', 'prefix\&suffix');
 ----
 prefixabcsuffix123
 
-query error invalid syntax for `regexp_replace`
+query error invalid digit found in string
 select regexp_replace('foobarbaz', 'b..', 'X', 1, 'g');
 
-query error invalid parameters specified in regexp_replace
+query error invalid digit found in string
```

Comment:
Are the changes precise?

Reply:
The error message in postgres is:
```
postgres=# select regexp_replace('foobarbaz', 'b..', 'X', 1, 'g');
ERROR:  invalid input syntax for type integer: "g"
```

It is similar to our current message `invalid digit found in string`. Both occurred during parsing the integer. But I didn't find where the error was generated in our code. Maybe we can improve the message by printing the invalid string.